examples/website/safegraph/package.json

```diff
@@ -11,8 +11,8 @@
   "dependencies": {
     "@loaders.gl/csv": "4.1.0-alpha.9",
     "d3-scale": "^2.0.0",
-    "deck.gl": "^8.8.0",
-    "mapbox-gl": "^1.13.0"
+    "deck.gl": "^9.0.0-alpha",
```

Comment:
nit: I'm not sure what `^9.0.0-alpha` does, shouldn't there be a number affixed at the end?

Reply:
Doesn't it mean any alpha? This is just to allow things to compile until we do the final release. Then it will be updated to `"^9.0.0"`

Reply:
I'm not sure... Based on https://semver.org/#spec-item-11 I believe `9.0.0-alpha` refers to a specific version number, with a sort order _before_ `9.0.0-alpha.1`. Normally `^` is defined as accepting any minor and patch releases on top of the specified version. Unclear to me how that is interpreted for an alpha. In any case, as long as `yarn install` isn't failing here, it's probably fine for now, agreed!